lib/ddtrace/contrib/sidekiq/server_internal_tracer/scheduled_push.rb

```diff
@@ -0,0 +1,21 @@
+# typed: true
+
+module Datadog
+  module Contrib
+    module Sidekiq
+      module ServerInternalTracer
+        # Trace scheduled job pushes
```

Comment:
Could you expand this documentation a bit?

When I read `scheduled job pushes` I think of pushing new jobs for Sidekiq to execute later, something that I would do in the "client" part of the application.

But because this class is under `ServerInternalTracer`, I'm not sure I understand what "scheduled job pushes" means here.

Reply:
Definitely a good call. I've updated this documentation, as well as the documentation for the heartbeat and job fetch tracing. Let me know if it's still unclear.